play/PlayerRules/Basic Rules.txt

```diff
@@ -570,29 +660,42 @@ unconsciousness. DC 50, Fortitude. Greater / Lesser poison increases damage
+Damage Types and armor types
 ==============================
 
-There are 4 main types of damage in compound X: Ballistic (conventional 
-firearms, bows), Laser, Plasma, and melee. Additionally, some Ballistic, Laser 
-and melee weapons can deal non-lethal variants. These damage types interact 
-differently with the two types of armour. Ballistic weapons and ballistic armour 
-are the first damage type and armour type players will come across. When 
-ballistic armour is shot by a ballistic weapon, the armour may apply some damage 
-reduction, then the player takes damage. The second type of damage players may 
-come across is plasma. Plasma damage stacks, and interacts fairly neutrally 
-with both ballistic and CS armours. Ballistic armour applies some DR to any 
-plasma damage, but in turn the plasma will melt away part of the armour. One 
-Tenth of the plasma's damage is removed from the armour's hit percent. For CS 
-shields, Plasma deals full damage to the shielding, plus inflicts plasma burn 
-on the wearer. Laser damage is the third type pf damage encountered by players. 
-All lasers deal burn damage on hit. Lasers are completely nullified by CS 
-shields, dealing all of their damage to the shielding, and inflicting no burn 
-damage. Lasers deal full damage to armoured targets, and remove nothing off of 
-the armour's hit percent. If a player wearing ballistic armour is hit with a 
-laser, they receive damage as if they weren't wearing any armour.
-
-Beyond weapon damage there are also environmental threats to consider. An 
+There are 5 main types of damage in compound X: Ballistic (conventional firearms, bows),
+Laser, Plasma, Explosive (concussive), and melee. Additionally, some Ballistic, Laser and 
+melee weapons can deal non-lethal variants. These damage types interact differently with 
+the two types of protection: Armor and Shields. 
+
+Ballistic Weapons:
+	Ballistic weapons are straightforward: they are blocked by armor and ignore shields.
+	Any damage that defeats armor damages health.
+Plasma Weapons:
+	Plasma interacts fairly neutrally with both armor and shields. Armor's effectiveness
+	is reduced by 1 APL when hit by plasma. Targets hit with plasma, that pierces armor, 
+	take (stackable) plasma burn. For Shields, Plasma deals half damage to the shielding,
+	plus inflicts plasma burn and any remaining damage on the wearer (only if their armor 
+	is pierced, of course). 
+Laser Weapons:
+	Lasers are completely blocked by shields, dealing all of their damage to the shielding.
+	Lasers deal full damage to armored targets; If a player wearing ballistic armor is hit 
+	with a laser, they receive damage as if they weren't wearing any armor.
+Explosives:
+	Certain grenades and devices have Explosive (concussive) damage. Explosive Damage ignores
+	armor (explosive damage is essentially, a crushing force). Fragmentation grenades are 
+	considered ballistic weapons, as their primary damage delivery method is shrapnel. 

```

Comment:
I'm hoping to give level 10ish players purchase access to better armors that target specific damage types. Such as EOD armor for explosives, ceramic/ablative/reflective armors for defending against lasers, etc. It would increase complexity, but players should have a very good understanding of gameplay by level 10, and it gives us a lot of creative options to add to the otherwise boring vanilla armor.